.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip pytest pytest-timeout
+        python -m pip -e .[dev] 
```

Comment:
This is missing the `install` clause, should be `python -m pip install -e .[dev]`

Reply:
Thanks, fixed